server/db/mysql/adapter.go

```diff
@@ -1132,9 +1132,9 @@ func (a *adapter) UserUnreadCount(uid t.Uid) (int, error) {
 // *****************************
 
 func (a *adapter) topicCreate(tx *sqlx.Tx, topic *t.Topic) error {
-	_, err := tx.Exec("INSERT INTO topics(createdat,updatedat,touchedat,state,name,owner,access,public,tags) "+
+	_, err := tx.Exec("INSERT INTO topics(createdat,updatedat,touchedat,state,name,usebt,owner,access,public,tags) "+
 		"VALUES(?,?,?,?,?,?,?,?,?)",
```

Comment:
The number of params doesn't appear to match: 10 vs 9

Reply:
done